package.json

```diff
@@ -11,8 +11,10 @@
     "reset": "npm run clean && npm ci"
   },
   "comments": {
-    "sentry": "@sentry/electron depends on @sentry/browser; use @sentry/browser as a transitive dependency to avoid version mismatches that make the build fail. See: https://github.com/getsentry/sentry-javascript/issues/1853",
-    "codrova-osx@6.0.0": "Version-controlled platform config files at apple/xcode/osx/Outline/config.xml, apple/xcode/osx/osx.json, and apple/xcode/osx/www/cordova_plugins.js as a workaround for https://github.com/apache/cordova-osx/issues/106. Delete these files when the issue is fixed."
+    "sentry":
```

Comment:
Can we do something about this? Every time we do npm it formats this back. Perhaps we can ignore `package.json` in the linter

Reply:
Done, I think, with `.clang-format-ignore`